pyproject.toml

```diff
@@ -24,8 +24,8 @@ dependencies = [
     "coverage>=7.6",
     "dagster>=1.9.9",
     "dagster-postgres>=0.25.9,<1", # Update when dagster-postgres graduates to 1.x
-    "dask>=2024",
-    "dask-expr>=1.1", # Required for dask[dataframe]
+    "dask>=2025",
+    "dask-expr>=2", # Required for dask[dataframe]
```

Comment:
Fixed some dependency weirdness that came up with the dependencies got re-locked after modifying `pyproject.toml`